### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by yichaoS**
added data updating process

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,15 +27,37 @@ Download the data files for a study folder, for example brca_tcga:
...
+```
 
+#### Step3 -Open a Pull Request on GitHub to the 'master' branch.
+For instructions on submitting a pull-request, please see:  [Using Pull Requests ](https://help.github.com/articles/using-pull-requests/) and [Sending Pull Requests](http://help.github.com/send-pull-requests/).
+
+## Automated tests on Circle CI
+All Pull Requests are automatically tested on [Circle CI](https://circleci.com/).
+
+Validation status of latest studies added to this repo: [![CircleCI](https://circleci.com/gh/cBioPortal/datahub.svg?style=svg)](https://circleci.com/gh/cBioPortal/datahub)
+
```
Comment (sandertan) [inline README.md]: Nice documentation! Some minor comments:

> All Pull Requests are automatically tested on [Circle CI](https://circleci.com/).

You can add that we use the the validation code from the cBioPortal master branch.

> Validation status of latest studies added to this repo:....

We just changed this! See the updated README in master :)

File: README.md [hunk 2]
```diff
@@ -32,8 +32,27 @@ Download the data files for a study folder, for example brca_tcga:
 git lfs pull -I public/brca_tcga
 ```
 
-## Download a complete MySQL export of the latest database
+## How to Upload Data
```
Comment (n1zea144) [inline README.md]: Is there anything on cloning the repos for starters?

**Comment by yichaoS** [comment]
@n1zea144 In the section of 'How to download data' there's some instructions for downloading either the zip files from portal or the whole repo from github

**Comment by yichaoS** [comment]
@sandertan updated to use the 'testing status' section from what's currently in the 'master' branch.

**Comment by n1zea144** [comment]
@yichaoS I mean, I didn't see it in your changes here, but looking at the larger README, I see its in the section before this - how to download data shows how to clone the repos.  Looks good to me.

**Pull request merged**